### PR TITLE
Break apart Streams docs

### PR DESCRIPTION
**Pull request by derrickdoo**
- Break apart Streams docs into 4 sections

### CODE REVIEW

File: includes/_footer.htm [hunk 1]
```diff
@@ -32,7 +32,8 @@
 				'streams',
 				'quickstart',
 				'toc',
-				'upgrade'
+				'upgrade',
+				'content'
```
Comment (guozhangwang) [inline includes/_footer.htm:36]: I found this name a bit misleading.. could we rename it to `streams-content` or `streams-subpages`?

**Review by guozhangwang — COMMENTED** [review]
One minor comment, otherwise LGTM.

File: includes/_footer.htm (hunk 1 above)
Comment (derrickdoo) [reply includes/_footer.htm:36]: I did this on purpose so we could have a generic name for containers that we know aren't going to rendered alongside themselves. It just makes it easier for other to copy and paste existing pages and build off of them.

Comment (guozhangwang) [reply includes/_footer.htm:36]: Got it, makes sense.

**Comment by guozhangwang** [comment]
Merged to `asf-site`.

**Pull request closed without merging**